### PR TITLE
Edit list category

### PR DESCRIPTION
**Pull request by pipinamngua**

### CODE REVIEW

**Comment by pipinamngua** [comment]
Edit list category

**Comment by pipinamngua** [comment]
ready

File: resources/views/user/products/product-list.blade.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+									</div>
+								</div>
+							</div>
+							<h5><a href="{{ route('show_pdetail', ['product' => $product->id]) }}">{{ $product->name }}</a></h5>
+							<div class="simpleCart_shelfItem">
+								<p class="flexisel_ecommerce_cart"><span>$312</span> <i class="item_price">{{ number_format($product->price_out, 2, '.', ',') }}</i></p>
+							</div>
+						</div>
+					</div>
+				</li>
```
Comment (trinhductoan) [inline resources/views/user/products/product-list.blade.php]: indent

File: app/Http/Controllers/GuestController.php [hunk 2]
```diff
@@ -33,11 +33,28 @@ public function getProductList($id)
...
         return view('user.checkout');
     }
+
+    public function getProductListSort($string)
+    {
+        $array = explode('-', $string);
+        $category = $array[0];
+        $sort = $array[1];
+
+        if ($sort == 1) {
```
Comment (trinhductoan) [inline app/Http/Controllers/GuestController.php]: viết thành scope

**Comment by trinhductoan** [comment]
commented

**Comment by pipinamngua** [comment]
fix bug

**Comment by pipinamngua** [comment]
ready

**Pull request merged**